CONTRIBUTING.md

```diff
@@ -29,6 +36,8 @@ Add your changes in commits [with a message that explains them](https://robots.t
 
 If you are adding code, make sure you've added and updated the relevant documentation and tests before you submit your pull request. Make sure to write tests that show the behavior of the newly added or changed code.
 
+Contributions should [follow](docs/standard-for-public-code.md) the requirements set out in the criteria of the Standard for Public code itself.
```

Comment:
Slightly more helpful for someone using a screenreader to skip from one hyperlink to another.

```suggestion
Contributions should [follow the requirements](docs/standard-for-public-code.md) set out in the criteria of the Standard for Public code itself.
```